gql/versions.go

```diff
@@ -124,9 +124,12 @@ func FinalizeVersionUploadAsync(ctx context.Context, mod *postgres.Mod, versionI
 	}
 
 	// TODO Validate mod files
```

Comment:
Should this TODO still be here?

Reply:
This is in current codebase as well, it can be removed.

Reply:
I'll remove it then in a commit.

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/0dcaad74bf3b745b8367792fb6fe0fc989a6ba23